AdyenCard/Form/DualBrandView.swift

```diff
@@ -90,6 +90,9 @@ extension FormCardNumberItemView {
             guard let firstLogo = logos.first else { return }
             let secondLogo = logos.adyen[safeIndex: 1]
             
+            primaryLogoUrl = firstLogo.url
```

Comment:
Tried to look commit by commit but I see these 2 lines were already in the code, why is it shown as newly added?

Reply:
This is my bad, I think I commited these two lines when was extracting this to a separate file 🙈

Reply:
It was like this before 
<img width="768" alt="image" src="https://github.com/user-attachments/assets/5ba46128-1580-46d7-8785-c047f1696563" />